src/services/auth.service.ts

```diff
@@ -11,6 +11,13 @@ export class AuthService {
 	static async whoAmI(): Promise<ServiceResponse<User>> {
 		try {
 			const { user } = await authClient.whoAmI({});
+			if (!user) {
+				throw new Error(
+					i18n.t("userNotFound", {
+						ns: "services",
+					})
+				);
+			}
```

Comment:
is this related to this pr?

Reply:
It's an optimization. 
We changed the user model (https://github.com/autokitteh/web-platform/pull/632/files#diff-20137314f228ed9053b19acb754e4a988a1b87972efb9eda79141369aa737c2b).

Before the change `user` wasn't mandatory, and we had to use conditional value, like this:
`protoUser?.data?.email`

Since the user model has been changed on our side, I optimized the code by a few lines to look better, like this:
`protoUser.email`

Reply:
Added the check to pass a user object which is definitely mandatory, and not `undefined` or `null`.